docs/ecosystem-user-event.mdx

```diff
@@ -117,7 +117,8 @@ to `await`!
 > Otherwise everything runs synchronously and you can ignore the promise.
 
 `type` will click the element before typing. To disable this, set the
-`skipClick` option to `true`.
+`skipClick` option to `true`. You will then need to focus the element
+yourself before calling `type`.
```

Comment:
Based on [this](https://github.com/testing-library/user-event/blob/master/src/__tests__/type.js#L42) comment in the type tests